Jenkinsfile

```diff
@@ -16,20 +16,23 @@ def baseConfig = {
 
 def defaultParams = [
     booleanParam(name: 'RELEASE_BUILD',
-        description: 'Is this a release build.',
+        description: 'Is this a release build. If so, GIT_REVISION must be specified, and only on-prem images will be built.',
         defaultValue: false),
     string(name: 'GIT_REVISION',
-        description: 'The git SHA to create the release build from.',
+        description: 'The git SHA to build ksqlDB from.',
+        defaultValue: ''),
+    string(name: 'CCLOUD_GIT_REVISION',
+        description: 'The cc-docker-ksql git SHA to build the CCloud ksqlDB image from.',
```

Comment:
i'm confused by this. the cloud build is downstream from this so why would you need to enter the git sha for cloud when building this? it seems like this makes the ksqldb on prem build dependent on the cloud ksql build.

Reply:
Also leftover from an earlier iteration. Removed.